packages/runtime/src/useCases/common/validation/TokenAndTemplateCreationValidator.ts

```diff
@@ -41,8 +43,37 @@ export class TokenAndTemplateCreationValidator<
                     );
                 }
             }
+
+            if (passwordProtection.passwordLocationIndicator && !TokenAndTemplateCreationValidator.isValidPasswordLocationIndicator(passwordProtection.passwordLocationIndicator)) {
+                validationResult.addFailure(
+                    new ValidationFailure(
+                        RuntimeErrors.general.invalidPropertyValue(
+                            `must be a number from 50 to 99 or one of the following strings: ${Object.values(PasswordLocationIndicatorOptions).slice(1).join(", ")}`
```

Comment:
`.slice(1)` will break as someone moves the items. Please filter out `RecoveryKit` instead.